index.js

```diff
@@ -182,11 +182,13 @@ for (p in plugins) {
 			var plugin = getMapValue(serviceProviderPlugins, spp);
 			// Skip GitHub plugins that do not match the run type
 			if (sp == 'github' && serviceProviderConfig.org && !plugin.org) continue;
-			if (compliance.includes(spp, plugin)) {
-				for (pac in plugin.apis) {
-					if (serviceProviderAPICalls.indexOf(plugin.apis[pac]) === -1) {
-						serviceProviderAPICalls.push(plugin.apis[pac]);
-					}
+			
+			// Skip if our compliance set says don't run the rule
+			if (!compliance.includes(spp, plugin)) continue;
```

Comment:
This was a merge conflict - this change reduces indentation and follows the pattern from above.